lib/config/ui_config.dart

```diff
@@ -213,9 +213,13 @@ abstract class DigitalGuideConfig {
       EdgeInsets.symmetric(vertical: 24, horizontal: 24);
   static const borderRadiusMedium = 8.0;
   static const heightSmall = 8.0;
+  static const heightMedium = 16.0;
   static const heightBig = 24.0;
   static const heightHuge = 48.0;
+  static const mediumButtonPadding =
+      EdgeInsets.symmetric(vertical: 8, horizontal: 14);
   static const paddingMedium = 16.0;
+  static const borderColor = Color(0x8077859D);
 }
```

Comment:
What about defining the color in `ColorConsts`?

Reply:
I can do it but I think it doesn't really matter, because we are using this color only once.

Reply:
So DigitalGuideConfig is not bad place for it because scope of it is much smaller than color consts.

Reply:
imo it can stay for now